pkg/csi/fuse/utils.go

```diff
@@ -41,3 +44,27 @@ func checkMountReady(fluidPath string) error {
 	glog.Infoln(string(stdoutStderr))
 	return err
 }
+
+func isMounted(absPath string) (bool, error) {
+	glog.Infof("abspath: %s\n", absPath)
+	_, err := os.Stat(absPath)
+	if err != nil {
+		return false, err
+	}
+
+	file, err := ioutil.ReadFile("/proc/mounts")
```

Comment:
Can this be replaced with mount.List()?

Reply:
I think so. I used the code from https://github.com/containers/build/blob/v0.4.0/util/mounts.go#L25 .

Reply:
Maybe we can enhance it in the next PR, because I've tested the current code for several cases.